docs/multicluster/user-guide.md

```diff
@@ -511,9 +511,10 @@ for more information.
 
 Since Antrea v1.9.0, Multi-cluster supports routing Pod traffic across clusters
 through Multi-cluster Gateways. Pod IPs can be reached in all member clusters
-within a ClusterSet. To enable this feature, the cluster's Pod CIDRs must be set in
-ConfigMap `antrea-mc-controller-config` of each member cluster like the example
-below. Note, **Pod CIDRs must not overlap among clusters to enable cross-cluster
+within a ClusterSet. To enable this feature, the cluster's Pod CIDRs must be set
+in ConfigMap `antrea-mc-controller-config` of each member cluster and make sure
```

Comment:
Probably use passive voice for both to consistent: "and `multicluster.enablePodToPodConnectivity` must be set to `true` in the `antrea-agent' configuration"

Reply:
Changed to passive voice.

Reply:
Then we should remove "make sure" too.

Reply:
Removed.